packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -40,7 +41,25 @@ test.serial(
 
     const materials = buildProposal(
       '@agoric/builders/scripts/fast-usdc/init-fast-usdc.js',
-      ['--oracle', 'a:agoric1watcher1'],
+      [
+        '--oracle',
+        'a:agoric1watcher1',
+        '--feedPolicy',
+        JSON.stringify({
+          nobleAgoricChannelId: 'channel-21',
+          nobleDomainId: 4,
+          chainPolicies: {
+            Arbitrum: {
+              cctpTokenMessengerAddress:
+                '0x19330d10D9Cc8751218eaf51E8885D058642E08A',
+              chainId: 42161,
+              confirmations: 1,
+              nobleContractAddress:
+                '0x19330d10D9Cc8751218eaf51E8885D058642E08A',
+            },
+          },
+        }),
+      ],
```

Comment:
Assuming you follow the `--net` suggestion for `init-fast-usdc.js`:

```suggestion
      ['--net'', 'MAINNET']
```

like https://github.com/Agoric/agoric-sdk/blob/6112a8b0fc5482e7930f5aa092a340dcef1a2a66/packages/boot/test/bootstrapTests/price-feed-replace.test.ts#L104

Reply:
Done. PTAL